tests/integration_tests/performance/test_net_tput.py

```diff
@@ -0,0 +1,276 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests the network throughput overhead added by Firecracker."""
+
+import json
+import time
+import statistics
+from subprocess import run, PIPE
+import pytest
+
+from retry import retry
+
+import host_tools.network as net_tools
+from framework.statistics import Statistics
+
+
+RUNS = 1
+
+IPERF = 'iperf3'
+
+# Iteration duration of a single run in seconds.
+ITERATION_DURATION = 1
+
+WINDOW_SIZE_MB = 10
```

Comment:
Not clear if you run tests without the specifying the windows size.

Reply:
I do, in the `run_iperf` method there is a for loop for window_size too which enables/disables the -w flag.